index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Survey Form</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <div id="head">
+        <h1 id=title>freeCodeCamp Survey Form</h1>
+        <p id="desc"><i>you for taking the time to help us improve the platform</i></p>
+    </div>
+    <div id="main">
```

Comment:
```suggestion
    <section id="main">
```

Please use HTML 5 semantic tags like (main, section, article) instead of using many div tags. So that you codes may look more clean and someone who takes a look on it may quickly recognise your codes easily.